src/arrays/quarray.jl

```diff
@@ -153,12 +153,12 @@
     typealias TupleArray{T<:Tuple,N} Array{T,N}
 
     # redudant definitions added to resolve ambiguity warnings
-    Base.getindex{B<:LabelBasis}(larr::DualVector{B}, tups::Union(Tuple,TupleArray)) = getindex(larr, bases(larr,1)[tups])
-    Base.getindex{B<:LabelBasis}(larr::CTranspose{B}, tups::Union(Tuple,TupleArray)...) = getindex(larr, map(getindex, bases(larr), tups)...)
+    Base.getindex{B<:LabelBasis}(larr::DualVector{B}, tups::@compat Union{Tuple,TupleArray}) = getindex(larr, bases(larr,1)[tups])
+    Base.getindex{B<:LabelBasis}(larr::CTranspose{B}, tups::@compat Union{Tuple,TupleArray}...) = getindex(larr, map(getindex, bases(larr), tups)...)
```

Comment:
You think it is this line? Try `@compat(Union{Tuple,TupleArray})...` instead ...